package.json

```diff
@@ -16,6 +16,7 @@
     "build-ci": "yarn global add node-gyp && yarn --ignore-engines && yarn build && yarn workspace commonwealth migrate-db",
     "chromatic": "chromatic --exit-zero-on-changes",
     "clean": "find . -name node_modules -exec rm -rf {} \\;; yarn --ignore-engines",
+    "dependency-graph": "npx depcruise packages/commonwealth/server.ts -T archi | dot -T svg > dependency-graph.svg",
```

Comment:
Interestingly, this breaks due to it needing node version >= 20. I have used madge before on our project to view dependency graphs of our front end. Anyways I will try fixing the errors and report back.

Reply:
This is just for local dev... to make sure we have a high level view of dependencies, but it can be improved. You need graphviz installed btw. On the other hand, upgrading to node 20 might be worth it.